circle.yml

```diff
@@ -536,6 +536,11 @@ jobs:
       - run:
           name: Test
           command: cargo test
+      - run:
+          name: Package
+          command: |
```

Comment:
Fancy way of doing this.
```suggestion
          working_dir: bindings/rust
          command: cargo package
```

Reply:
How can I leave a comment in that case? Just as a yaml comment?

Reply:
Yes. YAML commands are better IMHO.

Reply:
Is `~/bindings/rust` the current path? The suggested one does not work.